### PR TITLE
fixes #69755

### LINKED ISSUE #69755: Explorer: Regression - Focus Open Editors focuses on files, not editors
Issue Type: <b>Bug</b>

Insiders/OSS of 3-3-19

The focusOpenEditors command no longer focuses on the open editor 's list , rather it focuses on the files view.  Neither assigning a different key binding or the command pallet work.

VSC 1.31.1 performs correctly

Steps to reproduce,
- Issue focusOpenEditors from any context
- Explorer focuses on the files not the open editors

VS Code version: Code - Insiders 1.32.0-insider (804373ac4d5c84e96c8086cee5b9be132df88837, 2019-03-01T18:22:35.336Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 3696)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.87GB (17.44GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (31)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-javascript-snippet-pack|aka|0.1.5
Bookmarks|ale|10.2.2
pascal|ale|8.0.0
pascal-formatter|ale|2.1.0
git-easy|bib|1.11.0
bracket-pair-colorizer|Coe|1.0.61
vscode-eslint|dba|1.8.0
gitlens|eam|9.5.1
tslint|eg2|1.0.43
git-project-manager|fel|1.7.1
file-icons|fil|1.0.16
vscode-npm|fkn|3.3.0
code-runner|for|0.9.7
macros|ged|1.2.1
vscode-pull-request-github|Git|0.4.0
rest-client|hum|0.21.1
innosetup|idl|1.2.0
csharpfixformat|Leo|0.0.78
sftp|lix|1.9.5
start-git-bash|McC|1.2.1
vscode-remote-workspace|mkl|0.41.0
python|ms-|2019.2.5416
csharp|ms-|1.17.1
debugger-for-chrome|msj|4.11.2
vscode-icons|rob|8.2.0
git-add-remote|sam|0.0.2
code-settings-sync|Sha|3.2.5
vscode-hexdump|sle|1.7.2
gitblame|wad|2.6.3
JavaScriptSnippets|xab|1.7.2
markdown-all-in-one|yzh|2.1.0

(1 theme extensions excluded)

</details>

### PR DESCRIPTION
**Pull request by isidorn**
This PR basically reverts the original fix for https://github.com/Microsoft/vscode/issues/68101
The fix was that the focus needs to wait for the refresh to happen before focusing.
However since in the meantime I have fixed file events handling, this fix is no longer needed - the file will get picked up as deleted and will no longer exist in the tree.

This is good because that fixed caused https://github.com/Microsoft/vscode/issues/69755
Basically since we were focusing with a timeout the explorer was stealling focus from the open editors view.

I have verified that this indeed fixes 69755 and that 68101 remains fixed.

### CODE REVIEW

**Comment by isidorn** [comment]
Mergingin this PR to master. Will create a new off for release branch.
